src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -15,15 +17,26 @@ namespace NuGet.Services.EndToEnd.Support
 {
     public class V2V3SearchClient
     {
+        /// <summary>
+        /// In case cloud service information is not provided, this will be the fallback instance count.
+        /// </summary>
+        private const int DefaultServiceInstanceCount = 2;
```

Comment:
I removed InstanceCount from being provided in configuration. Now there are only 2 options:
1. Instance count is retrieved from Azure - will be the case for deployment scenarios
2. Default instance count (2) - for manual testing.